subprojects/kotlin-dsl/kotlin-dsl.gradle.kts

```diff
@@ -67,6 +68,7 @@ dependencies {
     testImplementation(project(":kotlinDslTestFixtures"))
     testImplementation(project(":buildCacheHttp"))
     testImplementation(project(":buildInit"))
+    testImplementation(project(":files"))
```

Comment:
I don't think it makes sense to have :files in implementation and testImplementation, does it? I would expect that everything in implementation is also in testImplementation.

Reply:
Done.